### PR TITLE
Streamline the e2e local tests to improve the pipeline execution time

### PR DESCRIPTION
**Pull request by mayank-s18**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -322,7 +322,7 @@ jobs:
             
             shopt -s globstar
             
-            SPECS=$(circleci tests glob "e2e_tests/features/**/*.feature" | circleci tests split --split-by=timings --show-counts)
+            SPECS=$(circleci tests glob "e2e_tests/features/*.feature" | circleci tests split --split-by=timings --show-counts)
```
Comment (DarrenW-HMPPS) [inline .circleci/config.yml:325]: Doesn't this mean that it won't run the tests in the features/e2e folder? Or any other folder that we choose to create in the future?

**Review by DarrenW-HMPPS — CHANGES_REQUESTED** [review]

File: .circleci/config.yml (hunk 1 above)
Comment (mayank-s18) [reply .circleci/config.yml:325]: The tests under features/e2e folder are currently skipped for e2e_local_test job. But even to skip these tests cypress spends a few mins on the pipeline to execute it and skip it. By excluding the features/e2e folder we should save those few mins cypress takes, hopefully.

Comment (DarrenW-HMPPS) [reply .circleci/config.yml:325]: I see.  That seems to be somewhat hidden and not obvious that it is happening.  Can we make it more explicit with appropriate folder hierarchy and naming?

Comment (mayank-s18) [reply .circleci/config.yml]: updated as requested..

File: .circleci/config.yml [hunk 2]
```diff
@@ -322,7 +322,7 @@ jobs:
             
             shopt -s globstar
             
-            SPECS=$(circleci tests glob "e2e_tests/features/**/*.feature" | circleci tests split --split-by=timings --show-counts)
+            SPECS=$(circleci tests glob "e2e_tests/features/e2eLocalTests.feature" | circleci tests split --split-by=timings --show-counts)
```
Comment (DarrenW-HMPPS) [inline .circleci/config.yml:325]: I think this might need to be e2e_tests/features/e2eLocalTests/*.feature

And similar for the one above

**Review by DarrenW-HMPPS — APPROVED** [review]

**Pull request merged**